<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Comunication</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>waiting</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OPEN PORT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ALIVE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GET_IR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MOTOR_T </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SERVO_T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GETDISTANCE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GETSPEED</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QTextEdit" name="state">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdIr_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IRLeft</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdDistance">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> DISTANCIA (CM)</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdIr_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>IR_right</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
